GpsDataCapturer/sharedlibrary/src/main/java/com/google/sharedlibrary/service/GpsDataCaptureService.java

```diff
@@ -124,7 +182,7 @@ public void onLowMemory() {
      */
     public void startCapture(LocationApiType locationApiType) {
         //create a new file
-        gpxFile = createGpxFile(gpxFileFolder, sdf.format(System.currentTimeMillis()));
+        gpxFile = createGpxFile(gpxFileFolder, createFileName());
 
```

Comment:
Change the format of filename from `yyyy-MM-dd'T'HH:mm:ss.SSS'Z' ` to `yyyy-MM-dd'T'HH-mm-ss.SSS'Z' `. The reason is the Mac system replace `:` with `/`, which causes problem while parsing the filename.